system/handlers/rest/auth/PresideApi.cfc

```diff
@@ -0,0 +1,26 @@
+component {
+	property name="configuredToken" inject="coldbox:setting:preside.restauthtoken";
+
+	private string function authenticate() {
+		var headers    = getHTTPRequestData( false ).headers;
+		var authHeader = headers.Authorization ?: "";
+		var token      = "";
+
+		try {
+			authHeader = toString( toBinary( listRest( authHeader, ' ' ) ) );
+			token      = ListFirst( authHeader, ":" );
+
+			if ( !token.trim().len() ) {
+				throw( type="missing.token" );
```

Comment:
I think rather than throwing here, we should be setting `restResponse.setStatusText( "Missing authentication token" );` See https://github.com/pixl8/preside-ext-data-api/blob/stable/handlers/rest/auth/DataApi.cfc, for example.

Basically, if we confirm that token is valid, return the token. Otherwise, return empty string and set the reason for failure in the restResponse.setStatusText()